### PR TITLE
chore: Removed Unused Variables

### PR DESCRIPTION
**Pull request by adithyaakrishna**
**Description:**

- Removed unused variables and added `// @ts-ignore` for needed ones

### CODE REVIEW

File: src/WebGL/Renderer.ts [hunk 1]
```diff
@@ -1797,10 +1799,12 @@ export class Renderer {
       b = 0,
       color,
       intensity,
+      // @ts-ignore
```
Comment (dkoes) [inline src/WebGL/Renderer.ts:1802]: what is the purpose of all the ts-ignore statement?

File: src/WebGL/Renderer.ts [hunk 2]
```diff
@@ -69,6 +69,7 @@ export class Renderer {
   // GL state cache
   private _oldDoubleSided = -1 as number | boolean;
   private _oldFlipSided = -1 as number | boolean;
+  // @ts-ignore
   private _oldBlending = -1;
```
Comment (adithyaakrishna) [inline src/WebGL/Renderer.ts:73]: Regarding, https://github.com/3dmol/3Dmol.js/pull/712#discussion_r1303000686, I wasnt sure if I should go ahead and remove unused variables in those places. Hence added // @ts-ignore. We can discuss it on our call today and finish this one up :)

File: src/WebGL/Renderer.ts (hunk 1 above)
Comment (adithyaakrishna) [reply src/WebGL/Renderer.ts:1802]: Just referenced the comment and addressed it here, https://github.com/3dmol/3Dmol.js/pull/712/files#r1303022588

File: src/WebGL/Renderer.ts (hunk 2 above)
Comment (adithyaakrishna) [reply src/WebGL/Renderer.ts]: Removed the unused variables and `// @ts-ignore` :)

**Pull request merged**